File: abc/abc172/b.cpp
// https://atcoder.jp/contests/abc172/submissions/27199836

#include <bits/stdc++.h>
using namespace std;
int main() {
  int ans = 0;
  string s, t; cin >> s >> t;
  for(int i = 0; i < s.size(); i++) if(s[i] != t[i]) ans++;
  cout << ans;
}

File: abc/abc172/b.java
// https://atcoder.jp/contests/abc172/submissions/43710819

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    String t = stdIn.next();

    int ans = 0;
    for(int i = 0; i < s.length(); i++) {
      if(s.charAt(i) != t.charAt(i)) {
        ans++;
      }
    }

    System.out.println(ans);
  }
}
